apps/web/pages/settings/my-account/profile.tsx

```diff
@@ -106,23 +106,16 @@ const ProfileView = () => {
   const updateProfileMutation = trpc.viewer.updateProfile.useMutation({
     onSuccess: async (res) => {
       await update(res);
-      // signout user only in case of password reset
```

Comment:
This is part of legacy code when user created an account using google, then changed the email which send password reset code, the flow is changed in this PR